python/lsst/cell_coadds/_stitched_psf.py

```diff
@@ -0,0 +1,162 @@
+# This file is part of cell_coadds.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+from functools import partial
+from typing import Any, Dict
+
+import lsst.geom as geom
+import numpy as np
+from lsst.afw.image import ImageD
+from lsst.afw.typehandling import StorableHelperFactory
+from lsst.meas.algorithms import ImagePsf
+
+from ._grid_container import GridContainer
+from ._uniform_grid import UniformGrid
+
+__all__ = ("StitchedPsf",)
+
+
+class StitchedPsf(ImagePsf):
+    """A piecewise PSF implementation backed by a 2-d grid of images."""
+
+    # We need to ensure a C++ StorableHelperFactory is constructed and
+    # available before any unpersists of this class.  Placing this "private"
+    # class attribute here accomplishes that.
+    _factory = StorableHelperFactory("lsst.cell_coadds", "StitchedPsf")
```

Comment:
Mypy will be happier if you annotate this with `ClassVar`; I think as it is it thinks it's a potential instance attribute with a default value.